org.knime.knip.ilastik/src/org/knime/knip/ilastik/nodes/headless/IlastikHeadlessNodeModel.java

```diff
@@ -388,8 +394,16 @@ private boolean runIlastik(final String tmpDirPath, final List<String> inFiles)
         writeToKnimeConsole(p.getInputStream());
         writeToKnimeConsole(p.getErrorStream());
 
+        while(p.waitFor(500, TimeUnit.MILLISECONDS)) {
+            exec.checkCanceled();
```

Comment:
should we handle the cancellation, e.g. by stopping the process and closing everything? At the moment we would just throw a `CanceledExecutionException` and we don't really handle it.